android/src/main/java/org/mozilla/mozstumbler/service/utils/TelemetryWrapper.java

```diff
@@ -1,14 +1,15 @@
 package org.mozilla.mozstumbler.service.utils;
 
 import android.util.Log;
-import org.mozilla.mozstumbler.service.AppGlobals;
+
+import org.mozilla.mozstumbler.svclocator.services.log.LoggerUtil;
+
```

Comment:
This formatting is what Android Studio does by default. I'm not sure how to change it, but I've included my code style settings as an importable JAR.